pkg/lib/cockpit-components-listing.jsx

```diff
@@ -334,8 +333,6 @@ export const Listing = (props) => {
     const bodyClasses = ["listing", "listing-ct"];
     if (props.fullWidth)
         bodyClasses.push("listing-ct-wide");
-    if (props.compact)
-        bodyClasses.push("listing-ct-compact");
```

Comment:
This class should then also be cleaned up from pkg/lib/listing.less. This can be a follow-up (until I find something else). Also, dropping the compact feature from the Listing component should be a separate commit, so that it's easier to cherry-pick into external branches.